xeofs/base_model.py

```diff
@@ -31,6 +32,8 @@ class BaseModel(ABC):
 
     """
 
+    extra_modules = []
```

Comment:
shouldn't this be a class attribute?
```suggestion
    self.extra_modules = []
```